### ISSUE TITLE
org.clojure/clojure not reported as out of date

### ISSUE THREAD

**Issue by lread** [liquidz/antq#55]
Given an empty directory with the following `deps.edn`
```Clojure
{:deps {org.clojure/clojure {:mvn/version "1.10.1"}
        org.clojure/clojurescript {:mvn/version "1.10.520"}}}
```

When I run antq in the same directory I get:
```Shell
> clojure -Sdeps '{:deps {antq/antq {:mvn/version "RELEASE"}}}' -M -m antq.core
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

|    :file |                     :name | :version | :latest-version |
|----------+---------------------------+----------+-----------------|
| deps.edn | org.clojure/clojurescript | 1.10.520 |        1.10.773 |
```

I expected to also see that Clojure 1.10.2 is available.

**Comment by liquidz** [comment]
@lread `org.clojure/clojure` is skipped by default, because it is common for me to specify an older version for testing multiple clojure versions.
https://github.com/liquidz/antq/blob/0.10.2/src/antq/core.clj#L66-L67
https://github.com/liquidz/antq/blob/0.10.2/deps.edn#L15-L16

But it is also good to provide a option not to skip clojure itself 👍 
(or you can use `--focus` option to force checking. e.g. `antq --focus=org.clojure/clojure`)

**Comment by lread** [comment]
Interesting! Thanks for the reply.

I specify multiple versions of Clojure for testing.  
But I always also include the latest version too, sometimes even a release candidate.

Would this maybe make sense?:
1. By default report if there is a newer version available than the latest version of `org.clojure/clojure`that I have specified in my deps.edn
2. Users can use the `--exclude` option if they want to suppress the check `org.clojure/clojure`.

**Comment by lread** [comment]
And also thanks for antq, I really enjoy using it! ❤️

**Comment by liquidz** [comment]
Make sense. Thanks!

Latest clojure version should be reported certainly.
I'll fix it.

**Comment by liquidz** [comment]
@lread I tried to fix in `dev` branch.
Could you try?

**Comment by lread** [comment]
Thanks @liquidz! I've given a sanity test over here on a few projects via:
```Shell
clojure -Sdeps '{:deps {antq/antq {:git/url "https://github.com/liquidz/antq.git" :sha "c730b6693a6fd4514c1672cf9c6ea661f8503197"}}}' -m antq.core
```
And it looks great to me!

I did notice a couple of things which are likely not directly related to this issue and may be me just not understanding implications/details.  Please take with a grain of salt:

**:override-deps ignored?**
One thing I did notice is that if I use `:override-deps` then `org.clojure/clojure` seems to be ignored.  For example:

```Clojure
{:deps {org.clojure/clojurescript {:mvn/version "1.10.520"}}
 :aliases {:other {:override-deps {org.clojure/clojure {:mvn/version "1.9.0"}}}}}
```
Antq only shows clojurescript:
```
|    :file |                     :name | :version | :latest-version |
|----------+---------------------------+----------+-----------------|
| deps.edn | org.clojure/clojurescript | 1.10.520 |        1.10.773 |
```
But if I change `:override-deps` to `:extra-deps` it is checked:

deps.edn
```Clojure
{:deps {org.clojure/clojurescript {:mvn/version "1.10.520"}}
 :aliases {:other {:extra-deps {org.clojure/clojure {:mvn/version "1.9.0"}}}}}
```
Antq returns:
```Shell
|    :file |                     :name | :version | :latest-version |
|----------+---------------------------+----------+-----------------|
| deps.edn |       org.clojure/clojure |    1.9.0 |          1.10.2 |
|          | org.clojure/clojurescript | 1.10.520 |        1.10.773 |
```
**Old versions of Clojure cause antq to fail**
Let's say I run antq against a Clojure 1.8.0 project (does that make sense, when was deps.edn introduced?).

deps.edn
```Clojure
{:deps {org.clojure/clojure {:mvn/version "1.8.0"}}}
```
antq will fail with a runtime exception:
```
Exception in thread "main" java.lang.RuntimeException: Unable to resolve symbol: qualified-symbol? in this context, compiling:(rewrite_cljc/node/token.cljc:13:9)
	at clojure.lang.Compiler.analyze(Compiler.java:6688)
	at clojure.lang.Compiler.analyze(Compiler.java:6625)
	at clojure.lang.Compiler$InvokeExpr.parse(Compiler.java:3766)
...
```

**Comment by liquidz** [comment]
@lread Thanks for confirmation!

> :override-deps ignored?

Yeah, I should add `:override-deps` here.
https://github.com/liquidz/antq/blob/0.10.2/src/antq/dep/clojure.clj#L38

> Old versions of Clojure cause antq to fail

Oh, nice catch!
`qualified-symbol?` is available since 1.9, and it seems rewrite-cljc-playground is using it.

> Unable to resolve symbol: qualified-symbol? in this context, compiling:(rewrite_cljc/node/token.cljc:13:9)

https://github.com/lread/rewrite-cljc-playground/blob/master/src/rewrite_cljc/node/token.cljc#L13

Do you have a plan to make rewrite-cljc-playground work with Clojure 1.8?

BTW rewrite-cljc-playground is great! Thanks!

**Comment by lread** [comment]
> Do you have a plan to make rewrite-cljc-playground work with Clojure 1.8?

Current plan is to support Clojure 1.9 and above.  
But if the community speaks, I will listen! 👂 

> BTW rewrite-cljc-playground is great! Thanks!

Thanks back at you for the cool work you do!
I am currently merging rewrite-cljc-playgound work into [clj-commons/rewrite-clj](https://github.com/clj-commons/rewrite-clj).

**Comment by liquidz** [comment]
@lread Thanks for confirmation!

I fixed `--upgrade` option only to work with Clojure 1.9.0 or later.
So checking outdated deps should work with Clojure 1.8.0.

`:override-deps` problem is also fixed in `dev` branch.

**Comment by lread** [comment]
Nice work @liquidz! I retried my tests above and all seems good now.

**Comment by liquidz** [comment]
Thanks!

Just released v0.10.3 #56
